src/components/home/home.jsx

```diff
@@ -80,6 +80,14 @@ class HomePage extends React.Component {
               </ul>
         </section>
 
+        <section id="home-volunteer">
+            <h1>Want to join the team?</h1>
+            <p>We welcome volunteers!</p>
```

Comment:
make this a subheading so that it looks different from the paragraphs below.

Reply:
@ambergkim Are you referring to the current <h1> here?